### PR TITLE
docker-context: add page

### PR DESCRIPTION
**Pull request by 7aske**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/docker-context.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# docker context
+
+> Lets you switch between contexts to manage multiple Docker environments.
```
Comment (acuteenvy) [inline pages/common/docker-context.md:3]: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood
> All descriptions must be phrased in the imperative mood.

```suggestion
> Switch between contexts to manage multiple Docker environments.
```

File: pages/common/docker-context.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`docker context create my-context`
+
+- Switch to a context:
+
+`docker context use my-context`
+
+- List all contexts:
+
+`docker context ls`
```
Comment (acuteenvy) [inline pages/common/docker-context.md:20]: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#placeholder-syntax
> User-provided values should use the `{{placeholder}}` syntax in order to allow `tldr` clients to highlight them.

https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming
> Use `snake_case` for multi-word placeholders.

```suggestion
- Create a context using a specific Docker endpoint:

`docker context create {{my_context}} --docker "host={{tcp://remote-host:2375}}"`

- Create a context based on the `DOCKER_HOST` environment variable:

`docker context create {{my_context}}`

- Switch to a context:

`docker context use {{my_context}}`

- List all contexts:

`docker context ls`
```

**Review by acuteenvy — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please read the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**